src/components/loader/loader.js

```diff
@@ -200,20 +202,34 @@ export default (() => {
      * @param {Object} plugins the config object with all plugins
      */
     addExternalPlugins(plugins) {
+      const externalPluginsPrecedence = typeof plugins.externalPluginsLoadPrecedence === 'boolean'
+        ? plugins.externalPluginsLoadPrecedence
+        : true
+      const externalPlaybacksPrecedence = typeof plugins.externalPlaybacksLoadPrecedence === 'boolean'
+        ? plugins.externalPluginsLoadPrecedence
+        : true
```

Comment:
It may have already been discussed, but I think using a more straightforward name would be better, like `loadExternalPluginsFirst` and `loadExternalPlaybacksFirst`. What do you think?
```suggestion
      const loadExternalPluginsFirst = typeof plugins.loadExternalPluginsFirst === 'boolean'
        ? plugins.loadExternalPluginsFirst
        : true
      const loadExternalPlaybacksFirst  = typeof plugins.loadExternalPlaybacksFirst === 'boolean'
        ? plugins.loadExternalPlaybacksFirst
        : true
```

Reply:
Another possibility would be instead of `before/after` (or first), indicate a preference. Since the plugin loading considers overrides, we could use a `preferExternalPlugins` and `preferExternalPlaybacks` as a hint.

Reply:
I think is important to explicit that these options only config the loading precedence because the overwrite feature still maintains the same behavior even with these options configured.

`loadExternalPluginsFirst` looks great! I will make the updates 🤙

Reply:
Done in this commit 5c95acc2e3cd7ed35d2e0c92c945525e33fe5117